README.md

```diff
@@ -519,6 +519,30 @@ func isError(error: Error?) -> Bool {
 }
 ```
 
+If there's a case where you want to enter a scope if AT LEAST ONE item is non-nil then use the `??` operator.
+
+```swift
+if let _ = item ?? item1 {
+	// At least one of these is not nil
+}
+```
+
+If you want to check for non-nil AND evaluate a bool property or series of bool properties, use the `where` keyword.
+
+```swift
+if let _ = item where propertyBoolOne && !propertyBoolTwo {
+	// Bool properties have been evaluated.
+}
+```
+
+If you want to enter a scope, only if an object is nil, then you can check directly for nil.
+
+```swift
+if item == nil {
```

Comment:
is that necessary? I think here you are just explaining how to use a feature, not defining a best practice, thoughts?